MartaMM-dataviz.md

```diff
@@ -0,0 +1,110 @@
+
+* data representation: marks and attributes
+### influencing factors:
+
+### potential applications of interactivity
+
+### use of colour
+<!---use of colour in domain-specific contexts: Christen, M., Brugger, P., & Fabrikant, S. I. (2021). Susceptibility of domain experts to color manipulation indicate a need for design principles in data visualization. PloS one, 16(2), e0246479. -->
+
+<!--### Storytelling
+Data visualisation and infographics are essential tools for _storytelling_-->
+
+### choosing the right visual representations for the data and task at hand
+### Anscombe's quartet
+### visual analysis as individual/social process
+
+In a sea of data, data visualisation represents a time-constrained activity which offers options to represent data analysis, making users/people aware of the different options they have. Dataviz can be considered an *informed list of possibilities* that empowers people instead of overwhelming them.
+tradeoff between having a large amount of information and knowing what to do with it 
+PEIR (personal environmental impact report)
+YFD (your flowing data): "to make personal data understandable to nonprofessional" ("Beautiful Data", Segaran and Hammerbacher, 2009).
+color schemes and other visual cues significantly help to convey the message behind the data representation.
+When collecting data through a form, it is crucial to tailor the form itself to the specific population it refers to (for example, using bigger font for older people).
+it is also crucial to design for _accurate_ data collection and maximise data quality. 
+test everything!
+different concerns with observational vs experimental data
+dataviz can, and should, help "data finds data" processes (see "Beautiful Data"). 
+
+### Importance of considering gender issues when approaching data and data visualisation
+"Users" are not all white men from a medium-high socioeconomic background, in a USA setting. 
+Male should *not* be the 'default' option, this attitude limits data accessibility, applicability, and generalizability. 
+As thoroughly described and argued by Caroline Criado Perez ("Invisible Women", 2019), taking gender data into account in (city/transport/local plan) planning requires changes and solutions that have a ripple effect with broader consequences on the general society.
+Fenwick, Sandell, and Berengueres (2019) in their book on data visualisation suggest a chart checklist to check for bias in data visualisation. This checklist covers aspects such as the type of chart, use of colurs, use of metaphors, and captions. 
+
+### Data accessibility
+
+How do you qualify what you produce in terms of data visualisation?
+<!--need to check what you wrote in the notes-->
+Use of colours and accessibility: [colororacle](www.colororacle.org) is a colour blindness simulator, allowing data scientists to consider colour branding and output restrictions when working on their charts.
+
+### Possible negative effects of data collection?
```

Comment:
I think in this part we can mention the misuse of data, for example, people giving the right of using their data for a purpose x but people actually using it for x, y, and z